### PR TITLE
Improvements to save and publish

### PR DESCRIPTION
**Pull request by seeta-ramaraju**
Changes related to Jira task - [G2P-218](https://www.ebi.ac.uk/panda/jira/browse/G2P-218)

### CODE REVIEW

File: src/components/curation/PublishSuccessAlert.vue [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default {
     <hr />
     <div class="d-flex justify-content-between">
       <router-link :to="`/lgd/${stableId}`" class="btn btn-primary me-3">
-        View New G2P Record
+        View New G2P Record {{ stableId }}
```
Comment (olaaustine) [inline src/components/curation/PublishSuccessAlert.vue:33]: Suggesting we use `View G2P Record {{ stableId }}` instead

File: src/views/AddLocusGeneDiseaseView.vue [hunk 2]
```diff
@@ -435,6 +479,7 @@ export default {
         </button>
         <button
           v-else
+          :disabled="isGeneDataLoading"
           type="button"
           class="btn btn-primary mb-3"
           @click="geneSearchBtnClickHandler"
```
Comment (olaaustine) [inline src/views/AddLocusGeneDiseaseView.vue:485]: Suggesting that we also use `@keyup-enter` to be able to use enter when a gene is entered. 
The `@keyup.enter="geneSearchBtnClickHandler"` needs to be added to L466

**Review by olaaustine — COMMENTED** [review]
Thank you @seeta-ramaraju  for this PR 
Still reviewing, just some minor changes for now

File: src/components/curation/PublishSuccessAlert.vue (hunk 1 above)
Comment (seeta-ramaraju) [reply src/components/curation/PublishSuccessAlert.vue]: taken care

File: src/views/AddLocusGeneDiseaseView.vue (hunk 2 above)
Comment (seeta-ramaraju) [reply src/views/AddLocusGeneDiseaseView.vue:486]: taken care

File: src/views/AddLocusGeneDiseaseView.vue [hunk 3]
```diff
@@ -305,39 +319,48 @@ export default {
...
+        this.saveBeforePublishErrorMsg =
           null;
-      this.isPublishSuccess = this.isSubmitSuccess = false;
-      this.isPublishDataLoading = this.isSubmitDataLoading = true;
+      this.isPublishSuccess = this.isSaveBeforePublishSuccess = false;
+
+      this.isSubmitDataLoading = true;
       this.stableId = null;
 
+      // Save and publish scenarios:
```
Comment (olaaustine) [inline src/views/AddLocusGeneDiseaseView.vue:354]: Please delete

**Review by olaaustine — COMMENTED** [review]
just suggesting we delete the scenario comment if possible

File: src/views/AddLocusGeneDiseaseView.vue (hunk 3 above)
Comment (olaaustine) [reply src/views/AddLocusGeneDiseaseView.vue:354]: Decided to keep it for future reference

**Pull request merged**